### DISCUSSION TITLE
Broken Link

### DISCUSSION THREAD

**Discussion by proJefferson** [MiKTeX/miktex#1074] [Q&A]
Hi, this link dont work.
https://linorg.usp.br/CTAN/systems/win32/miktex/setup/windows-x64/basic-miktex-21.12-x64.exe

Many users may find it difficult to download and then not be able to use the powerful tool and or worse go elsewhere and get a virus and tarnish Miktex's reputation
Whoever is responsible for maintaining the links could contact USP personnel.

**Comment by Aniol18** [comment]
It is best to download from the official website https://miktex.org/download and then there are no problems.
